SparkPost/2web2ui
# AC-1974 - Global tracking for profile and recipientLists forms

### What Changed

- Added form ids to allow list for all forms in: `2web2ui/src/pages/profile/*` and `2web2ui/src/pages/recipientLists/*`

### How To Test

- Verify that the new forms track in segment when submitted. 
- Verify that the form fields have the name and id attributes along with associated labels where needed. 

### To Do

- [x] Add test for reset 2fa tracking
- [x] Add test for disable 2fa tracking
- [x] Add test for generate 2fa backup codes
- [x] Add test for Edit Profile tracking
- [x] Add test for Password update tracking
